gqlgen.yml

```diff
@@ -35,3 +35,7 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.UIProjectFields
   User:
     model: github.com/evergreen-ci/evergreen/rest/model.APIUser
+  EventLogEntry:
+    model: github.com/evergreen-ci/evergreen/rest/model.APIEventLogEntry
+  EventLogData:
```

Comment:
The name for this should have the name "task" in it, since this event type is specific to tasks